src/main/java/com/fundation/webservice/model/CriteriaAudio.java

```diff
@@ -17,48 +17,60 @@
  */
 public class CriteriaAudio extends CriteriaConvert {
     private String newFormat;
-    private String aCodec;
-    private int aBit;
-    private int aChannel;
-    private int aRate;
+    private String audioCodec;
+    private int audioBit;
+    private int audioChannel;
+    private int audioRate;
 
+    /**
+     *
+     * Method getNewFormat receive the new format audio.
+     */
     public String getNewFormat() {
         return newFormat;
     }
 
+    /**
+     *
+     * Method setNewFormat establish the new format audio that is define in getNewFormat
+     */
     public void setNewFormat(String newFormat) {
         this.newFormat = newFormat;
     }
 
-    public String getaCodec() {
-        return aCodec;
+    /**
+     *
+     * Method getAudioCodec to work in the format audio required.
+     */
+    public String getAudioCodec() {
+        return audioCodec;
     }
 
-    public void setaCodec(String aCodec) {
-        this.aCodec = aCodec;
+    public void setAudioCodec(String audioCodec) {
```

Comment:
Comments please

Reply:
this documentation can review later please.